GOVERNANCE.md

```diff
@@ -0,0 +1,63 @@
+# Project Governance
+*Last updated June 9, 2022*
+
+## Overview
+
+FOSSBilling is an open source project which builds and maintains a free, open source billing and client management solution. 
+
+Anybody with an interest can join the community, contribute to the project, and be a part of the decision making process. In the next few paragraphs we will explain how that process takes place. 
+
+__TLDR; Project decisions are made by a consensus of the projects steering committee and informed by community discussion.__
+
+## Community
+
+The FOSSBilling community is made up of individuals, companies, or groups who use and/or contribute to the FOSSBilling software.
+
+Anybody can use the core FOSSBilling software within the terms of the [license](https://github.com/FOSSBilling/FOSSBilling/blob/main/LICENSE). There are not and never will be any fees for usage, and no special requirements.
+
+We really appreciate user contributions but if you just want to download, install, and go, then that is also fine. If you do want to be more involved in the community then [GitHub](https://github.com/FOSSBilling/FOSSBilling) and [Discord](https://fossbilling.org/discord) are good places to start. 
+
+## Steering Committee
```

Comment:
I'd appreciate a section here calling out how to become a contributor (contributor.md), but also how we assign permissions to new users.

ie. A user who demonstrate regular contributions to the project will be granted an appropriate level of access.

Reply:
NP, will add something.

Reply:
Added for role of Maintainer - We do not have any other documented 'levels' but we can add them to this document as and when we do.